js/data.js

```diff
@@ -552,12 +553,23 @@ var data = {
         dozent: ["Prof. Dr. Tobias Friedrich", "Dr. Timo Kötzing", "Dr. Andrew Sutton"],
         nameLV: "Heuristic Optimization"
     },
+    informationIntegration: {
+        kurz: "Information<br /> Integration",
+        kennung: [OSIS, SAMT],
+        lehrform: ["Vorlesung", "Übung"],
+        modul: ["Vertiefungsgebiete"],
+        semester: [ws15_16],
+        benotet: true,
+        cp: 6,
+        dozent: ["Prof. Dr. Felix Naumann"],
+        nameLV: "Information Retrieval"
+    },
     informationretrieval: {
         kurz: "Information<br /> Retrieval",
         kennung: [OSIS, IST],
         lehrform: ["Vorlesung", "Übung"],
         modul: ["Vertiefungsgebiete"],
-        semester: [ss14],
+        semester: [ss14, ws15_16],
```

Comment:
Please remove this.
Information Retrieval offers 6 LP this semester, so we need to make a new `informationretrieval6LP`.